server/lib/routes.js

```diff
@@ -89,6 +89,7 @@ module.exports = function (config, i18n) {
       '/settings/avatar/gravatar_permissions',
       '/settings/communication_preferences',
       '/change_password',
+      '/choose_what_to_sync',
```

Comment:
Can you add a server test for this in [tests/server/routes.js](https://github.com/mozilla/fxa-content-server/blob/master/tests/server/routes.js)